Makefile

```diff
@@ -10,7 +10,7 @@ GZIP ?= gzip -9 -n
 DIFFOSCOPE ?= diffoscope
 STRIP ?= strip
 
-CFLAGS ?= -O3
+CFLAGS ?= -O3 -g
```

Comment:
Maybe this should be set only when `DEBUG=1` is set?

Not sure if `-g` should be enabled for release builds too

Reply:
Typically you build with -g, create debug info package and strip the executable. But in this case the executable is tiny with debug symbols so it is bette to keep it simple and make crashes easier to debug.

Reply:
With debug info:

```
% ls -lh socket_vmnet
-rwxr-xr-x  1 nsoffer  staff    41K Nov  8 02:31 socket_vmnet
```

Without:

```
% ls -lh socket_vmnet
-rwxr-xr-x  1 nsoffer  staff    39K Nov  8 02:33 socket_vmnet
```

Reply:
I guess package maintainers want to consistently split the debug info for every package?

Reply:
Yes, this is done automatically by rpmbuild for example.